### DISCUSSION TITLE
Can't FlinkBatchSource be used with FlinkStreamSink?

### DISCUSSION THREAD

**Discussion by 7hong** [apache/seatunnel#1806] [Q&A] [answered]
Can't FlinkBatchSource be used with FlinkStreamSink? (such as KafkaTableStream and ClickHouse)

**Comment by ashulin** [comment, marked as answer]
Currently, SeaTunnel is not supported.

**Reply by 7hong** [reply]
Thank you for your answer. How to develop in the future? Will implement StreamAPI for ClickHouse?
